packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

```diff
@@ -8,7 +8,24 @@ title: Download and set up the SDK, Sign-In Widget, and sample app
 
 <StackSelector class="cleaner-selector"/>
 
-After you [create and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/), the next step is to download the GitHub repository that contains the SDK, Widget, and accompanying sample apps.
+This guide covers the next step after you [create and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/aspnet/main/), which is to download the GitHub repository that contains the SDK, Widget, and accompanying sample apps.
+
+---
+**Learning outcomes**
+* Configure the settings that you need to add to initialize the SDK and the Sign-In Widget.
+* Integrate the SDK and the Sign-In Widget into your own app after you run the sample app.
+
+**What you need**
+[Software Requirements](/docs/guides/oie-embedded-common-download-setup-app/android/main/#software-requirements).
+
+**Sample code**
+* Android - [okta-idx-android](https://github.com/okta/okta-idx-android)
```

Comment:
The hyphens in these lines should be em-dashes

Reply:
Fixed